src/ngraph/serializer.cpp

```diff
@@ -1649,7 +1649,7 @@ shared_ptr<Node> JSONDeserializer::deserialize_node(json node_js)
                                                                     groups);
             break;
         }
-        case OP_TYPEID::GRUCell:
+        case OP_TYPEID::GRUCell_v3:
```

Comment:
Don't change `serializer.cpp` -- the serializer will automatically use `visit_attributes` to do the serialization.

Reply:
I have removed `GRUCell case`  in `serializer.cpp` in favor of using `visit_attributes ` in the next commit.